src/components/map/overview-table.tsx

```diff
@@ -294,29 +299,44 @@ export function OverviewTable({
           <TableHeader>
             {table.getHeaderGroups().map((headerGroup) => (
               <TableRow key={headerGroup.id}>
-                {headerGroup.headers.map((header) => {
-                  return (
-                    <TableHead key={header.id}>
-                      <TooltipProvider>
-                        <Tooltip>
-                          <TooltipTrigger>
-                            {header.isPlaceholder
-                              ? null
-                              : flexRender(
-                                  header.column.columnDef.header,
-                                  header.getContext()
-                                )}
-                          </TooltipTrigger>
-                          <TooltipContent>
-                            {tooltips[
-                              header.column.id as keyof typeof tooltips
-                            ] || ""}
-                          </TooltipContent>
-                        </Tooltip>
-                      </TooltipProvider>
-                    </TableHead>
-                  );
-                })}
+                {headerGroup.headers.map((header) => (
+                  <TableHead key={header.id}>
+                    <TooltipProvider>
+                      <Tooltip>
+                        <TooltipTrigger>
+                          {header.isPlaceholder ? null : header.column.getCanSort() ? (
+                            <Button
+                              variant="ghost"
+                              onClick={header.column.getToggleSortingHandler()}
+                              className="h-max p-1 w-min w-full"
```

Comment:
This gives an error in Tailwind:
`'w-full' applies the same CSS properties as 'w-min'. (cssConflict)`

Which one is intentional?